mlflow/models/evaluation/default_evaluator.py

```diff
@@ -569,6 +569,15 @@ def plot_feature_importance():
             "shap_feature_importance_plot",
         )
 
+    def _evaluate_sklearn_model_score_if_scorable(self):
+        if self.model_loader_module == "mlflow.sklearn":
+            try:
+                score = self.raw_model.score(self.X, self.y)
+            except Exception as e:
+                _logger.warning(f"Computing sklearn model score failed, error {repr(e)}")
+                _logger.debug("", exc_info=True)
+            self.metrics["score"] = score
```

Comment:
I think this line fails if line 575 fails

Reply:
Let's add a test to make sure `_evaluate_sklearn_model_score_if_scorable` doesn't fail when `score` fails.